index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Abdul Ali Noori's portfolio" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <section>
+        <nav class="top-navigation">
+            <div class="welcome-message">Welcome</div>
+            <img src="images/Icons/Icon-Menu.svg" alt="GitHub">
+        </nav>
+        <header>
+            <div class="headline">
+                <p class="greeting-text">Hey there. I’m <br> Abdul Ali.</p>
```

Comment:
- [ ] Please replace this (`p `tag) with an (`h1` tag) instead since this is a new section so it denotes the heading of the section. If you want to read more about this you can check this [link](https://www.reliablesoft.net/h1-tag/) for more info. :100: